pom.xml

```diff
@@ -132,6 +132,16 @@
   </dependencies>
 
   <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <configuration>
+            <reuseForks>false</reuseForks>
```

Comment:
We are inserting a security provider which is static JVM config - so best to not attempt to reuse forks